django_socio_grpc/services.py

```diff
@@ -60,6 +95,14 @@ def before_action(self):
         """
         Runs anything that needs to occur prior to calling the method handler.
         """
+        if grpc_settings.GRPC_ASYNC:
```

Comment:
why doing all of that when you can just use sync_to_async in the upper context ?

Reply:
Because we want non blocking async permissions